tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh

```diff
@@ -81,8 +96,12 @@ cat > "$outdir"/"$name"/metadata/MANIFEST <<-EOF
   "minor-version" : "2",
   "patch-version" : "0",
   "configs"     : [ "$config" ],
-  "models"      : [ "$modelfile" ],
-  "model-types" : [ "$extension" ]
+  "models"      : [ $models_str ],
```

Comment:
It is a comment about `configs`, not `models`. I cannot point out `configs` line.
How can we set multiple configs? If we can pass multiple models, we also need to provide a way to pass multiple configs. There may be several options:

```
Option 1: 

$ model2nnpkg -c c1 c2 c3 -m m1 m2 m3

We need to update parsing logic. It requires an option before model files.

Option 2: 

$ model2nnpkg --start-config c1 c2 c3 --end-config m1 m2 m3

model2nnpkg does not support long option name.

Option 3:

$ model2nnpkg -c c1:c2:c3 m1 m2 m3

Option 4:

$ model2nnpkg -c c1:c2:c3 -m m1:m2:m3
```

Reply:
@glistening My preference: 1 > 4 > 3 > 2

Reply:
I also like option 1. At firs,t I thought it will break the usage, but we may add this option without breaking previous behavior for most cases (e.g. one model input only).

Reply:
I am okay for adding `-m` and `-c` options later in another PR. Let's merge this after making pr-checker happy. : )